packages/cli/lib/DevServerManager.js

```diff
@@ -26,7 +32,33 @@ class DevServerManager {
     }
   }
 
-  async start({ projectConfig, accountId, port }) {
+  generateURL(path) {
+    return this.path ? `${this.path}/${path}` : null;
+  }
+
+  async getProjectFiles(projectSourceDir) {
```

Comment:
Do we need this util? Looks like it doesnt do anything aside from call walk and is only used once